tutorials/machine-learning/plot_receptive_field.py

```diff
@@ -364,3 +341,8 @@
 plt.setp([iax.get_xticklabels() for iax in [ax1, ax2, ax3]], rotation=45)
 plt.autoscale(tight=True)
 mne.viz.tight_layout()
+
+###############################################################################
+# References
+# ==========
+# .. footbibliography::
```

Comment:
missing newline at end of file.  Our style checker will complain.  Other than that the changes look reasonable.

Reply:
Latest commit ( 287cf81 ) should fix that !